agr_literature_service/api/crud/utils/yml/reference.yml

```diff
@@ -0,0 +1,35 @@
+#
+#  NOTE: species has been left in here so that particular curie prefixes can be easily found
```

Comment:
are species still here ?

Reply:
Not species as such but at one point I thought only FB curators would be only able to add FB cross references and so we needed to have a way to determine who was allowed to add what (I thought FB curator would not be able to add an MGI cross ref, but people seemed happy with that). I left it in as that could change. Again its a simple change if people strongly feel it makes it confusing.